src/libraries/Microsoft.Extensions.Logging.Console/src/JsonConsoleFormatter.cs

```diff
@@ -141,6 +145,57 @@ private void WriteScopeInformation(Utf8JsonWriter writer, IExternalScopeProvider
             }
         }
 
+        private void WriteItem(Utf8JsonWriter writer, KeyValuePair<string, object> item)
+        {
+            var key = item.Key;
+            switch (item.Value)
+            {
+                case bool boolValue:
+                    writer.WriteBoolean(key, boolValue);
+                    break;
+                case byte byteValue:
+                    writer.WriteNumber(key, byteValue);
+                    break;
+                case sbyte sbyteValue:
+                    writer.WriteNumber(key, sbyteValue);
+                    break;
+                case char charValue:
+                    _singleCharArray[0] = charValue;
```

Comment:
Why do you need this?

Reply:
I wanted to convert the char to string avoiding the allocation.
Is there a better way to do it?
Or it is not worth to save the allocation here as we are allocating many strings anyway?

Reply:
@davidfowl suggested me to use `MemoryMarshal.CreateSpan` in another PR to achieve this, so I can get rid of the char array.
https://github.com/dotnet/aspnetcore/pull/22916#discussion_r464011195

However, I am not sure what I should do for the 4.6.1 target: should I use the array, simply create a string or else?

Reply:
Ok, I've removed it and used `ToString()` as fallback